package.json

```diff
@@ -29,7 +29,6 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "predeploy": "npm run build",
```

Comment:
@AnshulMalik Let this be there, for normal build processes which one could use during testing.